config/__init__.py

```diff
@@ -226,25 +225,23 @@ def get_variation(variation_key, variations_dict, defaults_dict):
 
 # Get AB experiment variation values from the environement.
 AB_VARIATIONS = {
-    'FORM_V1_VARIATION': os.environ.get('FORM_V1_VARIATION', '0 100'),
-    'FORM_V2_VARIATION': os.environ.get('FORM_V2_VARIATION', '100 100'),
+    'V1_VARIATION': os.environ.get('V1_VARIATION', '0 100'),
```

Comment:
@karlcow can you make sure to update the environment variables on the server to match this?

Reply:
(i did this)